fi.skyjake.Lagrange.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "fi.skyjake.Lagrange",
+    "default-branch": "release",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "lagrange",
+    "rename-desktop-file": "fi.skyjake.lagrange.desktop",
+    "rename-icon": "fi.skyjake.lagrange",
+    "finish-args": [
+        "--socket=x11",
```

Comment:
you also need to pass --share=ipc if you use --socket=x11 to allow x acceleration to work properly